android/src/main/java/io/sentry/react/RNSentryTimeToDisplay.java

```diff
@@ -0,0 +1,35 @@
+package io.sentry.react;
+
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.turbomodule.core.interfaces.TurboModule;
+
+import android.view.Choreographer;
```

Comment:
`l`: We could remove the unused imports
```suggestion
import com.facebook.react.bridge.Promise;

import android.view.Choreographer;
```